### PR TITLE
add SV section to VEP examples & use case page

### PR DESCRIPTION
**Pull request by sarahhunt**

### CODE REVIEW

File: docs/htdocs/info/docs/tools/vep/script/vep_example.html [hunk 1]
```diff
@@ -236,6 +236,43 @@ <h2 id="dbNSFP">dbNSFP</h2>
...
+  <p> <b>Prediction process</b>
+  <ul>
+  <li> The INFO keys 'END' or 'SVLEN' are present, the proportion of any overlapping feature covered by the variant is calculated
+  <li> If the SVTYPE or ALT is 'DEL', the variant tested for feature ablation/ truncation
+  <li> If the SVTYPE or ALT is 'DUP', the variant tested for feature amplification
+  <li> If the SVTYPE or ALT is 'INS' or 'DUP', the variant tested for feature elongatation
+  <li> SVTYPE is used in preference to ALT to derive the variant type of an SV with 'CN*' alleles
+  </ul></p>
+
+  <p><b> Reported overlaps</b>
```
Comment (ens-lgil) [inline docs/htdocs/info/docs/tools/vep/script/vep_example.html]: Same here

File: docs/htdocs/info/docs/tools/vep/script/vep_example.html [hunk 2]
```diff
@@ -236,6 +236,43 @@ <h2 id="dbNSFP">dbNSFP</h2>
...
+  <ul>
+  <li> VEP calculates the length and proportion of each genomic feature overlapped by a structural variant
+  <li> Use the <a href="vep_options.html#opt_overlaps">overlaps</a> option to enable this when using VCF or tab format.
+       (This is reported by default in standard VEP and JSON format.)
+  <li> The keys bp_overlap and percentage_overlap are used in JSON format and OverlapBP and OverlapPC in other formats.
+  </ul></p>
+
+  <p><b> Changing memory requirements</b>
```
Comment (ens-lgil) [inline docs/htdocs/info/docs/tools/vep/script/vep_example.html]: Same here

File: docs/htdocs/info/docs/tools/vep/script/vep_example.html [hunk 3]
```diff
@@ -236,6 +236,43 @@ <h2 id="dbNSFP">dbNSFP</h2>
...
   <hr/>
+
+
+  <h2 id="StructVar">Structural Variants</h2>
+
+  <p>VEP can be used to annotate structural variants with their predicted effect on other genomic features.
+     Input data should be supplied as VCF.
+  </p>
+
+  <p> <b>Prediction process</b>
```
Comment (ens-lgil) [inline docs/htdocs/info/docs/tools/vep/script/vep_example.html]: You could replace it by:
```
 <h4> Prediction process</h4>
```
and remove the `</p>` tag after the `</ul>` tag

File: docs/htdocs/info/docs/tools/vep/script/vep_example.html [hunk 4]
```diff
@@ -236,6 +236,43 @@ <h2 id="dbNSFP">dbNSFP</h2>
...
+  <li> If the SVTYPE or ALT is 'DEL', the variant tested for feature ablation/ truncation
+  <li> If the SVTYPE or ALT is 'DUP', the variant tested for feature amplification
+  <li> If the SVTYPE or ALT is 'INS' or 'DUP', the variant tested for feature elongatation
+  <li> SVTYPE is used in preference to ALT to derive the variant type of an SV with 'CN*' alleles
+  </ul></p>
+
+  <p><b> Reported overlaps</b>
+  <ul>
+  <li> VEP calculates the length and proportion of each genomic feature overlapped by a structural variant
+  <li> Use the <a href="vep_options.html#opt_overlaps">overlaps</a> option to enable this when using VCF or tab format.
```
Comment (ens-lgil) [inline docs/htdocs/info/docs/tools/vep/script/vep_example.html]: We used to display the options with the `--` in front of them, e.g. `--overlaps`

File: docs/htdocs/info/docs/tools/vep/script/vep_example.html [hunk 5]
```diff
@@ -236,6 +236,43 @@ <h2 id="dbNSFP">dbNSFP</h2>
...
+       (This is reported by default in standard VEP and JSON format.)
+  <li> The keys bp_overlap and percentage_overlap are used in JSON format and OverlapBP and OverlapPC in other formats.
+  </ul></p>
+
+  <p><b> Changing memory requirements</b>
+  <ul>
+  <li> By default, VEP does not annotate variants larger than 10M. If you are using the command
+  line tool, you can use the <a href="vep_options.html#opt_max_sv_length">max_sv_length</a> option to modify this.
```
Comment (ens-lgil) [inline docs/htdocs/info/docs/tools/vep/script/vep_example.html]: Actually this is `--max_sv_size`

File: docs/htdocs/info/docs/tools/vep/script/vep_example.html [hunk 6]
```diff
@@ -236,6 +236,43 @@ <h2 id="dbNSFP">dbNSFP</h2>
...
+  <li>By default, variants are analysed in batches of 5000. Using the <a href="vep_options.html#opt_buffer_size">buffer_size</a>
```
Comment (ens-lgil) [inline docs/htdocs/info/docs/tools/vep/script/vep_example.html]: Same remark, with the `--`

File: docs/htdocs/info/docs/tools/vep/script/vep_example.html [hunk 7]
```diff
@@ -244,31 +244,31 @@ <h2 id="StructVar">Structural Variants</h2>
...
-  </ul></p>
+  </ul>
 
-  <p><b> Changing memory requirements</b>
+  <h4> Changing memory requirements</h4>
   <ul>
   <li> By default, VEP does not annotate variants larger than 10M. If you are using the command
-  line tool, you can use the <a href="vep_options.html#opt_max_sv_length">max_sv_length</a> option to modify this.
-  <li>By default, variants are analysed in batches of 5000. Using the <a href="vep_options.html#opt_buffer_size">buffer_size</a>
+  line tool, you can use the <a href="vep_options.html#opt_max_sv_length">--max_sv_size</a> option to modify this.
```
Comment (ens-lgil) [inline docs/htdocs/info/docs/tools/vep/script/vep_example.html]: `#opt_max_sv_length` needs to be updated as well

**Comment by sarahhunt** [comment]
Thanks @ens-lgil !

**Pull request merged**